add waffle cheese bread recipe (#12)
cncf/cloud-native-community-cookbook: Recipies/waffle-cheese-bread.md

# **waffle-cheese-bread**

#### Recipe by Carlos Panato

> In Brazil the cheese bread (or the famous Pão de Queijo) is very know and it is delicious to eat as a snack

Reviewer: ```suggestion
In Brazil the cheese bread (or the famous Pão de Queijo) is very well known and it is delicious to eat as a snack.
```

> In Brazil the cheese bread (or the famous Pão de Queijo) is very know and it is delicious to eat as a snack

Reviewer: ```suggestion
In Brazil the cheese bread—the famous "pão de queijo"—is very well known and it is delicious to eat as a snack.
```

and together with a plain coffee is amazing.

> This version use the same base ingredients, but instead of you bake those bread we use the Waffle machine to cook it,

Reviewer: ```suggestion
This version uses the traditional base ingredients, but instead of it like bread, you cook it with a waffle maker instead.
```

> and that stay in the Waffle format.

Reviewer: ```suggestion
This way it stays in waffle format.
```

## INGREDIENTS

- 1 egg

> - 1 cup of milk (can use lactose free as well)

Reviewer: ```suggestion
- 1 cup of milk (lactose-free also works)
```

- 1/2 cup of oil
- Pinch of salt (can add black pepper if want to be more hardcore)
- 50g-100g of cheese (can use mozzarella, parmesan, you can mix)

> - If you love cheese can add a bit more, but that can be a bit greezy in the end

Reviewer: ```suggestion
- If you love cheese you can add a bit more, but it can turn out a bit greasy in the end
```

- 1 cup sour starch
- 1/2 cup sweet tapioca flour

## DIRECTIONS

> 1. Turn on your Waffle machine. Need to be very hot

Reviewer: ```suggestion
1. Turn on your Waffle machine; it needs to be very hot
```

> 2. Add all ingredients in the blender and mix it well

Reviewer: ```suggestion
2. Mix all ingredients in a blender
```
